packages/@sanity/base/src/components/ZIndexProvider.tsx

```diff
@@ -1,26 +1,26 @@
 import React, {createContext, useContext} from 'react'
-import vars from 'sanity:css-custom-properties'
+import cssCustomProperties from 'sanity:css-custom-properties'
 
 const defaults = {
   /*
     used by
     - Navbar
   */
-  navbar: parseInt(vars['--zindex-navbar'], 10) || 200,
-  navbarPopover: parseInt(vars['--zindex-navbar-popover'], 10) || 500000,
-  navbarDialog: parseInt(vars['--zindex-navbar-dialog'], 10) || 500001,
+  navbar: parseInt(cssCustomProperties['--zindex-navbar'], 10) || 200,
```

Comment:
Using parseInt here may obscure an error if the property doesn't exist on cssCustomProperties: parseInt will return NaN and the default value will be assigned instead. Perhaps make a separate function that ensures that the property exists and is a number or else throws?